oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
+enhancements or patches often don't need an OEP and can be injected into the Open
+edX development workflow with a patch submission.
+
+Step 2. Vet your idea
+~~~~~~~~~~~~~~~~~~~~~
+
+OEP Authors should first attempt to ascertain whether the idea is appropriate 
+for an OEP. Posting to the `open-edx-proposals Slack channel`_ is the best way
+to go about this.
 
 Vetting an idea publicly before going as far as writing an OEP is meant to save
-the potential author time. Many ideas have been brought forward for changing
-Open edX that have been rejected for various reasons. Asking the Open edX
-community first if an idea is original helps prevent too much time being spent
-on something that is guaranteed to be rejected based on prior discussions
-(searching the Internet is not always sufficient). It also helps to make
-sure the idea is applicable to the entire community and not just the author.
-Just because an idea sounds good to the author does not mean it will work for
-most Open edX installations.
-
-Initial Submission
-------------------
-
-Once the Author has asked the Open edX community whether an idea has any chance
+the potential authors time. Asking the Open edX community first if an idea is 
+original and appropriate helps prevent wasted effort. It also helps to make
+sure the idea is applicable to the entire community and not just the authors.
+
+.. _open-edx-proposals Slack channel: https://openedx.slack.com/messages/C1L370YTZ/details/
+
+Step 3. Create PR for "Draft" OEP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the Authors have asked the Open edX community whether an idea has any chance
 of acceptance, a draft OEP should be submitted as a pull request against the
-`central OEP repository`_. To identify the draft proposal, the Author should
+`central OEP repository`_. To identify the draft proposal, the Authors should
 check the numbered list of previous OEP pull requests and select the next 
-available number. The pull request title must start with "OEP-XXX", which 
-claims that OEP number for the included proposal.
+available number. The pull request title should be of the form 
+"OEP-XXXX-YYYY-ZZZZ", where
+
+* *XXXX* is the OEP number claimed for the included proposal
+* *YYYY* is the abbreviated type of the OEP (i.e., "proc", "bp" or "arch")
+* *ZZZZ* is a hyphenated brief (< 5 words) title of the proposal
 
 .. _central OEP repository: http://github.com/edx/open-edx-proposals
 
-After the Author drafts an OEP in a format in which they are comfortable, they
-will request an Arbiter from the edX Chief Architect. This Arbiter will be
-recorded in the "Arbiter" header on the OEP. The rest of Open edX community 
+Step 4. Request an Arbiter
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After the Authors draft an OEP in a format in which they are comfortable, they
+will request an Arbiter from the `edX Architecture Team`_ by emailing 
+`arch-team@edx.org`_. This Arbiter will be
```

Comment:
Maybe `edX Core Architecture Team` and `core-arch-team@edx.org` would better differentiate from the larger team of all architects?  Renaming proposals are imperfect, but you get the idea. :)

Reply:
For now, we've decided to keep this as is.  We may change it in the future as needed.